src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -61,9 +61,262 @@
 #include <beast/cxx14/memory.h> // <memory>
 #include <boost/foreach.hpp>
 #include <tuple>
+#include <ripple/basics/TestSuite.h>
 
 namespace ripple {
 
+namespace paging {
+    void
+    convertBlobsToTxResult (
+        std::vector<TxResult>& to,
+        std::uint32_t ledger_index,
+        std::string const& status, // the sql Status column
+        Blob const& txBlob,
+        Blob const& meta ) {
+
+        Serializer rawTxn (txBlob);
+        Serializer rawMeta (meta);
+        SerializerIterator it (rawTxn);
+        STTx::pointer txn = std::make_shared<STTx> (it);
+        auto tr = std::make_shared<Transaction> (txn, Validate::NO);
+
+        // TODO:
+        // tr->setStatus (status[0]);
+        tr->setLedger (ledger_index);
+
+        to.emplace_back(
+            std::make_pair(std::move(tr),
+                           std::make_shared<TransactionMetaSet> (
+                                tr->getID (), tr->getLedger (),
+                                    rawMeta.getData ())
+        ));
+    };
+
+    void
+    convertBlobsToTxResultHex (
+        std::vector<TxResultHex>& to,
+        std::uint32_t ledger_index,
+        std::string const& status, // the sql Status column
+        Blob const& txBlob,
+        Blob const& meta) {
+        to.emplace_back (
+            strHex(txBlob), strHex (meta), ledger_index);
+    };
+
+    static auto onUnsavedLedger = [] (std::uint32_t seq) {
```

Comment:
Not sure what this is for, but the code was handling the case of transactions with empty meta with this code. I moved it out, so I could write a test without needing an instantiated netops.